### PR TITLE
gitcollector support

### PR DESCRIPTION
**Pull request by lwsanty**
Signed-off-by: lwsanty <lwsanty@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+* /path/to/your-data-retrieval-util - a binary built locally.
+
+The repositories and downloaded/built binaries are cached by default in "repos" and "binaries" repositories from the current directory.
+
+
+Application Options:
+      --binaries=     Directory to store binaries (default: binaries) [$REG_BINARIES]
+      --repos=        Directory to store repositories (default: repos) [$REG_REPOS]
+      --url=          URL to the tool repo [$REG_GITURL]
+      --gitport=      Port for local git server (default: 9418) [$REG_GITPORT]
```
Comment (jfontan) [inline README.md:28]: Is `--gitport` needed or is a leftover from the previous tools? Same for `--repos-file` and `--complexity`.

Comment (jfontan) [reply README.md:28]: I've just checked that comes from `regression.Config`. We may need to update `regression-core` to have a base config and a `GitServerConfig`.

File: cmd/regression-retrieval/main.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+			if err.Type == flags.ErrHelp {
+				os.Exit(0)
+			}
+		}
+
+		log.Errorf(err, "Could not parse arguments")
+		os.Exit(1)
+	}
+
+	if options.Kind == "" {
```
Comment (jfontan) [inline cmd/regression-retrieval/main.go]: Setting a default in the option may be enough.

**Review by jfontan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lwsanty) [reply README.md:28]: agree

File: cmd/regression-retrieval/main.go (hunk 2 above)
Comment (lwsanty) [reply cmd/regression-retrieval/main.go]: Done.

**Review by mcarmonaa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lwsanty) [reply README.md:28]: https://github.com/src-d/regression-core/pull/19

**Comment by lwsanty** [comment]
let's block this one until I can refactor it according to https://github.com/src-d/regression-core/pull/19 changes

File: README.md (hunk 1 above)
Comment (lwsanty) [reply README.md:28]: done

**Comment by lwsanty** [comment]
done

**Review by jfontan — APPROVED** [review]

**Pull request merged**